spec/spec_helper.rb

```diff
@@ -2,11 +2,16 @@
 
 ENV['RAILS_ENV'] = ENV['RACK_ENV'] = 'test'
 
-require 'bootsnap'
-Bootsnap.setup(
-  cache_dir: "#{ENV.fetch('VENDOR_PATH', 'tmp')}/bootsnap",
-  development_mode: false
-)
+begin
+  require 'bootsnap'
+  Bootsnap.setup(
+    cache_dir: "#{ENV.fetch('VENDOR_PATH', 'tmp')}/bootsnap",
+    # development_mode: false
+  )
+rescue LoadError
+  # Bootsnap depends on ActiveSupport, but as AS heavily modifies stdlib
+  # we still want to test grape/sinatra without it
+end
```

Comment:
This is a problem in master, so maybe we want to add this to master instead of in this PR?

Reply:
Cherry picked the commit onto master as well 👍

Reply:
I guess Grape depends on AS too, so the comment should only read Sinatra.